pkg/cluster/controllers/storage/job.go

```diff
@@ -166,17 +175,17 @@ func (m *MinIOController) applyMinIOInitJob(ctx context.Context, harborcluster *
 	return minioUnknownStatus(), nil
 }
 
-func (m *MinIOController) checkMinIOInitJobReady(ctx context.Context, harborcluster *goharborv1.HarborCluster) (bool, error) {
+func (m *MinIOController) checkMinIOInitJobReady(ctx context.Context, harborcluster *goharborv1.HarborCluster) (*batchv1.Job, bool, error) {
 	job := &batchv1.Job{}
 	if err := m.KubeClient.Get(ctx, m.getMinIONamespacedName(harborcluster), job); err != nil {
-		return false, err
+		return nil, false, err
 	}
 
 	for _, condition := range job.Status.Conditions {
 		if condition.Type == batchv1.JobComplete {
-			return condition.Status == corev1.ConditionTrue, nil
+			return job, condition.Status == corev1.ConditionTrue, nil
 		}
 	}
 
-	return false, nil
+	return job, false, nil
 }
```

Comment:
This function could be moved to [github.com/goharbor/harbor-operator/pkg/resources/statuscheck](https://github.com/goharbor/harbor-operator/tree/master/pkg/resources/statuscheck).